activerecord/test/cases/attribute_methods_test.rb

```diff
@@ -1234,6 +1237,15 @@ def subject_was
     assert_equal("overridden_subject_was", obj.subject_was)
   end
 
+  test "#alias_attribute with an overridden original method along with an overridden alias method in a parent class doesn't issue a deprecation" do
+    obj = assert_not_deprecated(ActiveModel.deprecator) do
```

Comment:
It made me realized that we end up using wrong deprecator - https://github.com/rails/rails/pull/48917/files